src/templates/pycontw-2020/_includes/staff.html

```diff
@@ -12,13 +12,28 @@ <h2>{% trans 'Registration' context 'staff' %}</h2>
 </ul>
 <h2>{% trans 'Development' context 'staff' %}</h2>
 <ul class="staff-list">
-	<li></li>
+	<li>Tom Chen</li>
+	<li>tai</li>
+	<li>Wei-Hsiang (Matt) Wang</li>
+	<li>Rex</li>
+	<li>阿嘉</li>
+	<li>Alice</li>
+	<li>Macs</li>
+	<li>Maliao Guo</li>
+	<li>Annie Bu</li>
+	<li>Pearlie</li>
+	<li>GTB</li>
+	<li>Jeremy Sung</li>
+	<li>Leon Fu</li>
+	<li>Josix</li>
+	<li>SHY</li>
+	<li>Fliss</li>
```

Comment:
He recently changed his name to `Flynn`. Please help to update this. thx!
```suggestion
	<li>Flynn</li>
```